docs/using/frameworks/flask.rst

```diff
@@ -0,0 +1,196 @@
+Flask
+=====
+
+The oso Flask integration provides a more convenient interface to oso for
+usage with Flask.
+
+Usage
+-----
+
+Initialization
+~~~~~~~~~~~~~~
+
+The :py:class:`flask_oso.FlaskOso` class is the entrypoint to the integration.
```

Comment:
```suggestion
The :py:class:`~flask_oso.FlaskOso` class is the entrypoint to the integration.
```
The tilde only shows the last name, which cleans things up. Would be nice to use throughout.

Reply:
Thanks great idea.